src/string/memcmp.c

```diff
@@ -0,0 +1,20 @@
+#include <string.h>
+
+int
+memcmp(const void *s1, const void *s2, size_t n)
+{
+	size_t i = 0;
+	const char *cs1, *cs2;
+	cs1 = (const char *)s1;
+	cs2 = (const char *)s2;
+	for (; i < n; i++)
+	{
+		char first = cs1[i];
+		char second = cs2[i];
+		if (first != second)
```

Comment:
why this two variables ? they are uneeded, i is also uneeded (you can do `if (*cs1 != *cs2)`)